test/integ/pandas/test_arrow_pandas.py

```diff
@@ -742,7 +741,8 @@ def test_empty(conn_cnx):
         df_count = 0
         for _ in cursor.fetch_pandas_batches():
             df_count += 1
-        assert df_count == 0
+        # an empty df is yielded
+        assert df_count == 1
```

Comment:
This means that this would be a behavior change.
I fired up the latest master code and it does look like `fetch_pandas_batches` returns an empty iterator for empty results, while `fetch_pandas_all` returns an empty `pandas.DataFrame` with the column names.
Let's adhere to this old behavior